packages/phone-number-privacy/common/src/utils/config-utils.ts

```diff
@@ -1,5 +1,21 @@
 import BigNumber from 'bignumber.js'
+import * as dotenv from 'dotenv'
+
+dotenv.config()
 
 export const toNum = (value: BigNumber.Value) => new BigNumber(value).toNumber()
 export const toBool = (value: string | undefined, fallback: boolean) =>
   value ? value.toLowerCase() === 'true' : fallback
+
+export function fetchEnv(name: string): string {
+  if (process.env[name] === undefined || process.env[name] === '') {
+    throw new Error(`ENV var '${name}' was not defined`)
+  }
+  return process.env[name] as string
+}
+
+export function fetchEnvOrDefault(name: string, defaultValue: string): string {
+  return process.env[name] === undefined || process.env[name] === ''
```

Comment:
```suggestion
  return process.env[name] || process.env[name] === ''
```

Reply:
Did you mean to put '!' in front of process.env? I think we might want to be careful in case of env variables with the value 'false' or '0'. I *think* js always treats these as strings so probably fine.